neptune/new/exceptions.py

```diff
@@ -1004,19 +1004,18 @@ def __init__(self, location: str, namespace: str):
 
 
 class NeptuneFeaturesNotAvailableException(NeptuneException):
```

Comment:
I think the proper naming is in single form Neptune**Feature**~s~NotAvailableException